app/src/main/java/co/condorlabs/customcomponents/customspinner/SpinnerFormField.kt

```diff
@@ -121,6 +118,13 @@ class SpinnerFormField(
         (autoCompleteTextView?.adapter as? SpinnerFormFieldAdapter)?.replaceStates(data.sortedBy { it.label })
     }
 
+    fun setItemSelectedHint(){
```

Comment:
Why not just setHint?

Reply:
setHint means that you are going to set a value to the hint ,
with setItemSelectedHint() I preserve the structure of the other function with a similar behavior